api/templates/rest_framework_swagger/base.html

```diff
@@ -0,0 +1,100 @@
+{% load staticfiles %}
```

Comment:
@samanehsan We found that this refactor of the templates is present in the latest development version of RFS. So it'd be good to have a comment here about removing when we upgrade to the next release.